components/brave_wallet/browser/zcash/zcash_serializer.cc

```diff
@@ -23,55 +21,32 @@ namespace {
 constexpr char kTransparentHashPersonalizer[] = "ZTxIdTranspaHash";
 constexpr char kSaplingHashPersonalizer[] = "ZTxIdSaplingHash";
 constexpr char kOrchardHashPersonalizer[] = "ZTxIdOrchardHash";
-constexpr char kTxHashPersonalizerPrefix[] = "ZcashTxHash_";
+
+// https://zips.z.cash/zip-0244#txid-digest-1
+constexpr uint32_t kConsensusBranchId = 0xC2D6D0B4;
+constexpr char kTxHashPersonalizer[] =
+    "ZcashTxHash_"
+    "\xB4\xD0\xD6\xC2";
 
 constexpr uint32_t kV5TxVersion = 5 | 1 << 31 /* overwintered bit */;
 // https://zips.z.cash/protocol/protocol.pdf#txnconsensus
 constexpr uint32_t kV5VersionGroupId = 0x26A7270A;
-constexpr uint32_t kConsensusBranchId = 0xC2D6D0B4;
-
-// https://zips.z.cash/zip-0244#txid-digest-1
-std::string GetTxHashPersonalizer() {
```

Comment:
@cypt4 should probably review changes in components/brave_wallet/browser/zcash/zcash_serializer.cc here.